platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -399,6 +399,15 @@ public void start() {
         platformWiring.startGossip();
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @NonNull
+    @Override
+    public KeysAndCerts getKeysAndCerts() {
```

Comment:
Wasn't this provided to the platform builder? https://github.com/hashgraph/hedera-services/blob/1f6a473b4c550e20d7b1bf104196b95bc146cdcd/hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java#L331

We should not be adding new dependencies from the App to Platform for information that the app has before the platform.

Reply:
Makes sense, I'll update this.